test/functional/ios/appium_tests.py

```diff
@@ -100,11 +97,9 @@ def test_hide_keyboard_presskey_strategy(self):
         self.assertFalse(el.is_displayed())
 
     def test_hide_keyboard_no_key_name(self):
-        el = self.driver.find_element_by_name('Uses of UITextField')
-        el.click()
+        self._move_to_textbox()
 
-        # get focus on text field, so keyboard comes up
-        el = self.driver.find_element_by_class_name('UIATextField')
+        el = self.driver.find_elements_by_ios_predicate('wdType == "XCUIElementTypeTextField"')[0]
```

Comment:
why not to use find_element?

Reply:
Explicitly to show more than one `XCUIElementTypeTextField`.
That's it, behavior is same to `find_element`.
(If you have further comments on my idea, it's welcome.)